.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
```

Comment:
```suggestion
  push:
    branches:
      - master
```

Reply:
Sure, I can change it like this if you prefer. Just know that the deployment happens only on master anyway also with the current version. Personally, I would not recommend restricting the tests to only run on master (if you're concerned about gh-pages, it won't run there either, because it doesn't have the github workflow file).

Reply:
This will still run CI on branches which have been submitted as PRs. I just don't want to run CI on random branches which are neither master nor PRs.